website/src/plugins/malva.md

```diff
@@ -25,7 +25,7 @@ dprint config add g-plane/malva
 
 This will update your config file to have an entry for the plugin. Then optionally specify a `"malva"` property to add configuration:
 
-```jsonc
+```json
```

Comment:
I think these need to be jsonc because of the comment below? I forget if the website handles it right or not

Reply:
If we set it `jsonc`, syntax highlighting and version replacement will be broken.